docs/background/gardener_hibernation.md

```diff
@@ -0,0 +1,47 @@
+---
+description: How hibernation of Gardener-based Kubernetes clusters works
+---
+
+# Putting Gardener clusters in hibernation
+
+In {{brand}}, you have the option of putting a Gardener-based Kubernetes
+[cluster in hibernation](../howto/kubernetes/gardener/hibernate-shoot-cluster.md).
+This is something you might want to do whenever you know you won't be
+needing the cluster for some time, so putting it in hibernation makes
+sense because from that time on and before you wake it up, you pay
+*less* for hosting.
+
+You may wonder why you still pay *something* after putting a cluster in
+hibernation, instead of paying nothing at all. To answer this question,
+we have to explain how hibernation actually works.
+
+## How hibernation works
+
+When you put a Gardener cluster in hibernation, what really happens is
+that all worker nodes are destroyed. So for as long as the cluster is in
+hibernation, you will not be paying for CPU, RAM, and boot volume
+storage utilization incurred by the worker nodes. But there might still
+be resources created by the cluster, for which you will keep getting
+charged even when the cluster is in hibernation. More specifically, you
+will keep paying for any persistent volumes, floating IPs, and load
```

Comment:
Explain how these come into being. (Kubernetes PVCs, services with an external IP, and services of `Type: LoadBalancer`.)